zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -0,0 +1,569 @@
+                raise Exception('failed to found snapshot for lun[%s]' % lun_name)
+            for info in ret.result.fileInfo:
+                if snapshot_name in info.fileName:
+                    seq_num = info.seqNum
+                    break
+
+            install_path = FORMAT_CBD_SNAPSHOT_PATH.format(zbsutils.get_physical_pool_name(logical_pool_name), logical_pool_name, lun_name, str(seq_num))
+        else:
+            install_path = cmd.installPath
+
+        start_port, end_port = linux.parse_port_range(cmd.portRange)
+        port, l = linux.find_free_port_with_locking(start_port, end_port)
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)